AmericanMedicalAssociation/AMA-style-guide
# Ama 89 news featured organism

https://palantir.atlassian.net/browse/AMA-89

Adds the news featured list organism.

### Steps to Test:

1. Navigate to Organisms > Sections > News Featured
2. Observe that the News Featured organism is present, with the News Label molecule on the left, and a news featured row item on the right.
